local_setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# Use a .env file with default settings
+if ! ls .env; then
```

Comment:
I would use `! test -f .env` instead and then it won't print the error message from `ls` which may be a little confusing to users